container/go/cmd/loader/loader.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2015 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/////////////////////////////////////////////////////////////////////
+// This rule imports an image from a `docker save` tarball using the go_containerregistry.
+//
+// Extracts the tarball, examines the layers, and creates a container_import
+// target for use with container_image
+
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/bazelbuild/rules_docker/container/go/pkg/oci"
+	tb "github.com/google/go-containerregistry/pkg/v1/tarball"
+	"github.com/pkg/errors"
+)
+
+var (
+	directory = flag.String("directory", "", "Where to save the image's files.")
```

Comment:
Call this variable and flag `outDir` to be more explicit that it's an output directory.